roles/lib_openshift/src/ansible/oc_adm_registry.py

```diff
@@ -0,0 +1,47 @@
+# pylint: skip-file
+# flake8: noqa
+
+def main():
+    '''
+    ansible oc module for registry
+    '''
+
+    module = AnsibleModule(
```

Comment:
The serviceaccount is the preferred approach, since it removes the maintenance required to maintain the certs for the embedded kubeconfig, and there are no concerns with api server hostnames, etc.